docker-compose.yaml

```diff
@@ -1,27 +1,30 @@
 version: "2.2"
 services:
   api:
-    image: camptocamp/swissgeol_local_api:latest
-    init: true # handle kill signals for rust
+    build:
+      context: ./api
+      dockerfile: Dockerfile
+      args:
+        - ENV=development
+      target: builder
     ports:
       - 8480:3000
     environment:
       AWS_HOST: http://minio:9000
       AWS_ACCESS_KEY_ID: &minio_user minio
       AWS_SECRET_ACCESS_KEY: &minio_pass minio123
-      ENVIRONMENT_NAME: local-dev
       PGUSER: &db_user www-data
       PGPASSWORD: &db_pass www-data
       PGDATABASE: &db_name swissgeol-local
       PGHOST: db
       PGPORT: "5432"
       APP_PORT: "3000"
-      SQLX_OFFLINE: "true"
-    command: ["cargo", "watch", "--shell", "cargo run --offline --target x86_64-unknown-linux-musl"]
+      DATABASE_URL: "postgres://www-data:www-data@db:5432/swissgeol-local"
```

Comment:
Is there a way to not duplicate the db infos?

Reply:
Addressed in https://github.com/swissgeol/ngm/pull/827/commits/64d2e491ea8b53a42bd1f8ab13bee9b45c89a357